src/components/ManaCalculator/hooks/useManaState.ts

```diff
@@ -153,6 +153,15 @@ export function useGivenManaState(
     setState({
       ...state,
       userType: value,
+      [getStakedOrDelegated(value)]: state.heldTokens,
```

Comment:
why is this new key pair needed?

Reply:
when changing a user, I feel like it's best to preserve and sync the value from the owned amount (`heldTokens`) to the `staked/delegated amount` input. This is what this does